svelte-motoko-starter/README.md

```diff
@@ -0,0 +1,190 @@
+<p align="left" >
+  <img width="240"  src="./assets/logo.png">
+</p>
+
+# Svelte Dapp with Motoko & Internet Identity
+
+This repository is meant to give Svelte developers an easy on-ramp to get started with developing decentralized applications (Dapps in short) for the Internet Computer blockchain. Dapps, also known as smart contracts are specialized software that run on a blockchain.
+
+This template contains
+
+- a Svelte frontend app under `src/frontend` to be hosted on-chain, with support for authentication using Internet Identity
+- a Mokoto dapp under `src/backend` to serve as a backend to the Svelte frontend
+
+You can see a deployed version of this template here: https://zixfv-4yaaa-aaaam-aaatq-cai.ic0.app/
+
+### What is the Internet Computer?
+
+The Internet Computer is a novel blockchain that has the unique capability to serve web content while not requiring the end users to use a browser extension, such as Metamask.
+
+Coupled with super fast execution the Internet Computer provides the worlds first truly user friendly Web 3.0 experience.
+
+### What are canisters?
+
+Dapps on the Internet Computer live in canisters, which are special smart contracts that run WebAssembly, and can respond to regular HTTP requests, among other capabilities.
+
+This repository uses Svelte for the frontend running in the browser with Mokoto running the business logic of your dapp, and will build and deploy multiple _canisters_:
+
+- `backend` that is written in Motoko, and will hold the business logic of your dapp.
+- `frontend` that is your regular Svelte app, transferred into an `frontend asset` canister.
+- `internet_identity` that this repository uses as an authentication provider. It is written in Rust.
+
+### What is Motoko?
+
+Motoko is a new language designed for the Internet Computer. Easy to learn for JavaScript and Solidity developers. Created by the Motoko team at the DFINITY Foundation, led by WebAssembly co-creator Andreas Rossberg. To learn more about the language, check out the [SDK](https://smartcontracts.org/docs/language-guide/motoko.html).
+
+### What is Internet Identity?
+
+This starter template also includes integration with Internet Identity. Internet Identity is a new authentication framework similar to Github or Google login, but providing complete anonimity to the users. To learn more about Internet Identity check out the [documentation](https://smartcontracts.org/docs/ic-identity-guide/what-is-ic-identity.html).
+
+## Install dependencies
+
+Make sure you have [node.js](https://nodejs.org/) installed.
+
+### How to get this template
+
+To clone this template without downloading the entire repository, run the following command:
+
+```
+npx degit olaszakos/examples/svelte-motoko-starter svelte-motoko-starter
```

Comment:
I suppose this has to be adjusted for this PR?

Reply:
Thank you, fixed